teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<ul><li><strong>thresholds of only 10, 5, and 5</strong><u>, respectively</u><strong>.</strong></li><li>we will scale all property values to an&nbsp;<strong>scaled threshold of 5. </strong>-&gt; maybe: ... scale them to fit in the <strong>range [0,5]?</strong></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>We do not - we only use a scaling factor that will map 500 to 5 and 600 to 6. I could not find a name for this kind of scaling.</p>

Reply:
<p>@t-kimber, maybe you have a better idea of how to phrase this? </p><p> </p>